src/Swift.Runtime/src/TypeDatabase.xml

```diff
@@ -2,46 +2,46 @@
 <swifttypedatabase version="1.0">
```

Comment:
Do we plan to use this format for type database snapshots? Do we need more information here?

I would prefer not to create a specific format if all the information can be included in the generated managed bindings.

Reply:
I think we need to encode the information about blittability and frozeness in the TypeDatabase. We need to know whether a type coming from a different module is frozen or not.

Reply:
We need to define format of the snapshot and let's explore if we can emit all information needed in the generated assembly. I will add it to the tracking issue, out of scope for this PR.